bootstrapvz/providers/ec2/tasks/ebs.py

```diff
@@ -8,7 +8,12 @@ class Create(Task):
 
     @classmethod
     def run(cls, info):
-        info.volume.create(info._ec2['connection'], info._ec2['host']['availabilityZone'])
+        # EBS volumes support encryption. KMS key id is optional and default key
+        # is used when it is not defined.
+        encrypted = info.manifest.data['provider']['encrypted'] if 'encrypted' in info.manifest.data['provider'] else False
```

Comment:
Instead of this rather long statement, you should use `get()` with a default, like so:  
```
encrypted = info.manifest.data['provider'].get('encrypted', False)
```